src/Deserialization/UnspentOutput.purs

```diff
@@ -95,12 +97,13 @@ convertValue value = do
           :: Array (ScriptHash /\ Array (AssetName /\ BigNum))
     -- convert to domain types, except of BigNum
     multiasset'' :: Map T.CurrencySymbol (Map T.TokenName BigNum) <-
-      Map.fromFoldable <$>
```

Comment:
Why has this been changed to `notImplemented`?

Reply:
It stopped compiling, I marked it to come back later and forgot. Sorry. Fixed now.